### ISSUE TITLE
Multiple tabs / open link in new tab

### ISSUE THREAD

**Issue by maackle** [ssbc/patchwork#705]
I'm definitely missing the ability to check out another message, user, channel, etc., without losing my place in the current thread I'm reading.

If a core dev could weigh in and give an estimate of the feasibility/difficulty of implementing this, I might be able to work on this (depending on the answer).

**Comment by mmckegg** [comment]
Open in new window would be the simplest.

A few things would need to be extended in `index.js`, and `main-window.js` would need a way of externally setting its location `app.navigate`. But otherwise, everything else is already in place. 

Tabs would be possible, but I really don’t want to add this to patchwork as it makes the user interface much more complicated and there is no good way of doing multiprocess tabs in electron. The user interface would also have to be redesigned considerably.

**Comment by maackle** [comment]
@mmckegg OK, I dove into this with the goal of opening new windows. Without any real Electron knowledge, I decided to naively replicate the functionality of `lib/window.js`, but using `electron.remote.BrowserWindow` instead of `electron.BrowserWindow`, and blindly pulling in the minimal configuration to make something happen. (I would definitely DRY up an actual PR but I'm just trying to get something working.)

The window opens, but the JS that gets executed in the new window can't `require('mutant')`. Any ideas? Is this a decent approach or should I not be using `electron.remote` for this?

This is what I've tried so far: https://github.com/maackle/patchwork/commit/6cd5da1b4ddd1cb2af671df4d146795d02f88267#r26505308
